v2/conn.go

```diff
@@ -52,10 +53,11 @@ type Conn struct {
 }
 
 // NewConn creates a new Conn used for server. On client side, use Dial instead.
-func NewConn(laddr net.Addr, counter uint8) *Conn {
+func NewConn(laddr net.Addr, counter uint8, ifTypes []uint8) *Conn {
```

Comment:
I think the type of the last argument should be `...uint8` to make it simpler for callers.

Reply:
Done